ui/pages/developer_page.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+class DeveloperPage:
+    def __init__(self, driver):
+        self.driver = driver
```

Comment:
it's better to create a base class for all pages to keep the construction logic consistent

```suggestion
import logging

class BasePage:
    __url__: str

    def __init__(self, driver):
        self.driver = driver
        self.wait = WebDriverWait(self.driver, 10)
        self.url = self.__url__
        self.logger = logging.getLogger(self.__class__.__name__)
        
class DeveloperPage(BasePage):
    __url__ = "https://neonevm.org/developers"
```

Reply:
Fixed